### PR TITLE
Improved the clock_time_get module

### LINKED ISSUE #24: Investigate if wasm module examples are practical / Make the clock_time_get example Wasm Module return the the time.
Currently, this just prints out the string "Done!". Which was perfect for when we were testing / building the wasm-trasnformer, but now it does not make much sense for the docs 😄 

relates to https://github.com/wasmerio/docs.wasmer.io/issues/20#issuecomment-575625369

### PR DESCRIPTION
**Pull request by torch2424**
closes #24 

cc @ChrisWhealy 

This introduces a more practical example for the `clock_time_get` example, using AssemblyScript, and the fixes some links for the in progress multi-value return for `clock_time_get` 😄 

# Examples

Screen Shot 2020-01-29 at 3 59 24 PM
Screen Shot 2020-01-29 at 3 59 45 PM
Screen Shot 2020-01-29 at 4 00 14 PM

### CODE REVIEW

**Comment by MaxGraey** [comment]
@torch2424  btw you could avoid heap allocation:
```ts
const getTimeResponseBuffer: i32[] = [0]; // this moved in static data section and don't use heap allocation

let statusCode = clock_time_get(0, 1000, getTimeResponseBuffer.dataStart);

...

let clockResponse = unchecked(getTimeResponseBuffer[0]);
```

**Comment by torch2424** [comment]
@MaxGraey

Yo! Thanks for the suggestion 😀 So since it's for a small example, more on the general Wasi binding than AssemblyScript, and I'm focusing on the readability in this example, I think I'll keep the `_alloc` version. But I super appreciate the suggestion 😀

File: docs/wasmer-js/module-transformation.md [hunk 1]
```diff
@@ -41,16 +39,41 @@ Ok, back in reality...
...
-In order to understand whether or not this module needs transformation, we need to take a look inside the WebAssembly module.
+In order to understand whether or not this module needs transformation, we need to take a look at the WebAssembly module.
 
-> ### ASIDE
-> We make no attempt to teach you WebAssembly here!
->
-> If you want to know about the inner workings of a WebAssembly module, then please visit the [WebAssembly.org](https://webassembly.org) website and read the documentation there.
->
-> We now continue with your scheduled program...
+In the Transforming WASI Modules examples, there is an example [AssemblyScript](https://github.com/AssemblyScript/assemblyscript) module, in which is uses the [clock_time_get WASI binding from the wasi_unstable phase](https://github.com/AssemblyScript/assemblyscript/blob/899e97ab28a857da07a533551eb937d771677f93/std/assembly/bindings/wasi_unstable.ts#L36). For example:
```
Comment (ChrisWhealy) [inline docs/wasmer-js/module-transformation.md]: A couple of somewhat pedantic points here...

* Grammatical use of the passive voice.  The above sentence should read   
    "...example AssemblyScript module, in which the clock_time_get WASI binding from the wasi_unstable phase ***is used***"  
    😀

* Use of previously un-explained terminology.  
    Is it reasonable for a non-expert to understand what is meant by the term "*wasi_unstable phase*"?  
    May I suggest that this term is either not used, or given a brief explanation

Thanks

**Comment by syrusakbary** [comment]
Based on the last examples, this is no longer needed.

Thanks anyway for the work @torch2424.

**Comment by torch2424** [comment]
No worries! Thanks @syrusakbary :smile: :+1:

**Pull request closed without merging**